rpc/events.go

```diff
@@ -26,7 +26,7 @@ type EventsChunk struct {
 
 type EmittedEvent struct {
 	*Event
-	BlockNumber     *uint64    `json:"block_number,omitempty"`
-	BlockHash       *felt.Felt `json:"block_hash,omitempty"`
```

Comment:
Events emitted from pending blocks dont have them, so we need to mark them `omitempty`.